scripts/check_if_pr_is_low_risk_test.py

```diff
@@ -213,22 +216,23 @@ def mock_print(unused_str):
         self.assertListEqual(diff_files, [])
         self.assertDictEqual(file_diffs, {})
 
-    def test_parse_diff_failure_no_diff(self):
+    def test_parse_diff_failure_no_diff(self) -> None:
 
-        def mock_run_cmd(tokens):
+        def mock_run_cmd(tokens: List[str]) -> str:
             if '--name-status' in tokens:
-                return (
+                cmd_string = (
                     'M       modified\n'
                 )
             if tokens[-1] == 'modified':
-                return (
+                cmd_string = (
                     'diff --git a/modififed b/modified\n'
                     'index 11af605ef2b7..89d00105ca66 100644\n'
                     '--- a/modified\n'
                     '+++ b/modified\n'
                 )
+            return cmd_string
```

Comment:
Previously I was doing this because whenever we define a return type of a function then MyPy expects a return value in every case, but since we are returning in only 2 cases MyPy throwing error `Missing return statement`. So, to fix this issue I did these changes.

But, Now it is fixed by returning the None value at the end. PTAL and let me know if you have comments.